SampleApplications/SDK/Opc.Ua.Client/Session.cs

```diff
@@ -857,6 +868,7 @@ public int GoodPublishRequestCount
                  endpointDescription,
                  endpointConfiguration,
                  clientCertificate,
+                 configuration.SecurityConfiguration.SendCertificateChain? clientCertificateChain : null,
```

Comment:
>configuration.SecurityConfiguration.SendCertificateChain [](start = 17, length = 56)

move compare to line 854, only load chain if flag is set

Reply:
fixed.